### PR TITLE
test initialization arguments

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

**Comment by ashtul** [comment]
@mnunberg 
This is to make testing init args in rmtest.
Done it the long way on [RedisBloom](https://github.com/RedisBloom/RedisBloom/pull/112).

**Comment by ashtul** [comment]
@mnunberg 
I have made an effort to put the arguments also in the function so I don't need a class to every each and every one of them but I couldn't figure it out. If it ain't too much effort, it would be cool. (but then we kind of retiring it anyway...)

File: rmtest/__init__.py [hunk 1]
```diff
@@ -106,6 +106,19 @@ def assertExists(self, r, key, msg=None):
...
+            c, s = self.client, self.server
+        except Exception:
+            delattr(self, '_server')
+            self.assertOk('OK')
+        else:
+            self.assertOk('NotOK')  
+
+    def assertInitArgsSuccess(self):
+        c, s = self.client, self.server
+        self.assertOk('OK', self.cmd('set', 'test', 'foo'))
```
Comment (mnunberg) [inline rmtest/__init__.py]: ping/ping is more appropriate here; it's better not to have tests which modify data impicitly

**Review by mnunberg — CHANGES_REQUESTED** [review]

File: rmtest/__init__.py (hunk 1 above)
Comment (ashtul) [reply rmtest/__init__.py]: @mnunberg 
Great point!
It does return True/OK though, not 'pong'. Probably parsing somewhere along the way.

**Review by mnunberg — APPROVED** [review]

**Comment by ashtul** [comment]
Thanks @mnunberg !
I have no writing rights here. Can you merge it in?

**Pull request merged**